### PR TITLE
[#1073] Document the changed command endpoint prefix

### PR DESCRIPTION
**Pull request by calohmn**
This is for #1073:
Document the changes to the Command & Control endpoint names.

### CODE REVIEW

File: site/documentation/content/concepts/command-and-control.md [hunk 1]
```diff
@@ -53,24 +53,28 @@ that was opened by the application. If the response reached the application, the
...
 
 The response of the command will be sent by the device to 
 
-* `control/[${tenant}]/[${device-id}]/res/${req-id}/${status}`
+* `command/[${tenant}]/[${device-id}]/res/${req-id}/${status}`
 
 If the device is authenticated, the `${tenant}` and `${device-id}` are left empty (resulting in 3 subsequent `/`s).
 
+{{% note %}}
+Previous versions of Hono used `control` instead of `command` as topic prefix. Using the `control` prefix is still supported but deprecated.
```
Comment (sophokles73) [inline site/documentation/content/concepts/command-and-control.md]: I think we can omit this note from the concept page ... it is sufficient to have this in the API spec FMPOV

File: site/documentation/content/concepts/command-and-control.md [hunk 2]
```diff
@@ -85,8 +89,12 @@ The following diagrams show the message flow for commands over the MQTT adapter:
 
 When a device connected to the AMQP adapter wants to receive commands from the adapter, it opens a receiver link specifying the following source address:
 
-* `control` for authenticated devices
-* `control/${tenant}/${device-id}` for unauthenticated devices
+* `command` for authenticated devices
+* `command/${tenant}/${device-id}` for unauthenticated devices
+
+{{% note %}}
+Previous versions of Hono used `control` instead of `command` as address prefix. Using the `control` prefix is still supported but deprecated.
```
Comment (sophokles73) [inline site/documentation/content/concepts/command-and-control.md]: I think we can omit this note from the concept page ... it is sufficient to have this in the API spec FMPOV

File: site/documentation/content/concepts/command-and-control.md (hunk 1 above)
Comment (calohmn) [reply site/documentation/content/concepts/command-and-control.md]: Ok, I've removed it.

File: site/documentation/content/concepts/command-and-control.md (hunk 2 above)
Comment (calohmn) [reply site/documentation/content/concepts/command-and-control.md]: Ok, I've removed it.

**Pull request merged**